### PR TITLE
[dagster-fivetran][docs] Migrate Fivetran docs to new doc site

### PR DESCRIPTION
**Pull request by maximearmstrong**
## Summary & Motivation

Copy and migrate new Fivetran docs to new site, see original PR #26026

Removing `docs-to-migrate` tag from the original PR

## How I Tested These Changes

BK.

### CODE REVIEW

**Comment by maximearmstrong** [comment]
* **#26485** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26485?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/26485?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: docs/docs-beta/docs/integrations/libraries/fivetran.md [hunk 1]
```diff
@@ -18,19 +18,65 @@ tags: [dagster-supported, etl]
 ---
 
 
+# Using Dagster with Fivetran
```
Comment (cmpadden) [inline docs/docs-beta/docs/integrations/libraries/fivetran.md:21]: h1 header can be removed in `docs-beta` as it's derived from `title` front matter.

File: docs/docs-beta/docs/integrations/libraries/fivetran.md [hunk 2]
```diff
@@ -18,19 +18,65 @@ tags: [dagster-supported, etl]
...
+<details>
+  <summary>Prerequisites</summary>
 
-<CodeExample filePath="integrations/fivetran.py" language="python" />
+- The `dagster` and `dagster-fivetran` libraries installed in your environment
+- Familiarity with asset definitions and the Dagster asset graph
+- Familiarity with Dagster resources
+- Familiarity with Fivetran concepts, like connectors and connector tables
+- A Fivetran workspace
+- A Fivetran API key and API secret. For more information, see [Getting Started](https://fivetran.com/docs/rest-api/getting-started) in the Fivetran REST API documentation.
```
Comment (cmpadden) [inline docs/docs-beta/docs/integrations/libraries/fivetran.md:40]: This is showing _undefined_
image

File: docs/docs-beta/docs/integrations/libraries/fivetran.md [hunk 3]
```diff
@@ -18,19 +18,65 @@ tags: [dagster-supported, etl]
...
+
+</details>
+
+## Represent Fivetran assets in the asset graph
+
+To load Fivetran assets into the Dagster asset graph, you must first construct a <PyObject module="dagster_fivetran" object="FivetranWorkspace" /> resource, which allows Dagster to communicate with your Fivetran workspace. You'll need to supply your account ID, API key and API secret. See [Getting Started](https://fivetran.com/docs/rest-api/getting-started) in the Fivetran REST API documentation for more information on how to create your API key and API secret.
```
Comment (cmpadden) [inline docs/docs-beta/docs/integrations/libraries/fivetran.md:46]: The `PyObject` components are creating an _undefined_. Is that specific to Fivetran or more broadly an issue with the new docs? If more broad we can merge and make a follow-up ticket to fix this.
image

Comment (maximearmstrong) [reply docs/docs-beta/docs/integrations/libraries/fivetran.md:46]: I get the same PyObject not implemented yet error in the I/O manager guide that is already in the docs

In the I/O manager guide:
Screenshot 2024-12-16 at 2 42 04 PM

In the Fivetran docs:
Screenshot 2024-12-16 at 2 43 17 PM

Comment (maximearmstrong) [reply docs/docs-beta/docs/integrations/libraries/fivetran.md:46]: cc @neverett

File: docs/docs-beta/docs/integrations/libraries/fivetran.md (hunk 1 above)
Comment (maximearmstrong) [reply docs/docs-beta/docs/integrations/libraries/fivetran.md]: Fixed in [d82cf47](https://github.com/dagster-io/dagster/pull/26485/commits/d82cf4702a85eb3a643a564963a333a4780aa091)

File: docs/docs-beta/docs/integrations/libraries/fivetran.md (hunk 3 above)
Comment (neverett) [reply docs/docs-beta/docs/integrations/libraries/fivetran.md:43]: Yeah, I agree with @cmpadden -- let's merge this, and I'll make a ticket to follow up on `PyObject` implementation in new docs.

**Review by neverett — APPROVED** [review]

**Pull request merged**